packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql

```diff
@@ -18,7 +16,14 @@ query FetchLoggedInUser {
     projects {
       total
       results {
-        ...projectFragment
```

Comment:
Hm. The idea behind this was to always have these two in sync so we avoid an additional fetch. A bit frustrating the tooling forces us to be "simple" here at a risk.

Reply:
Yeah, I still don't know exactly what changed but that's the only solution I could find so far that would work.